benchmarking/commons/benchmark_definitions/transformer_wikitext2.py

```diff
@@ -26,41 +25,40 @@
 )
 
 
-def transformer_wikitext2_default_params(sagemaker_backend: bool) -> Dict[str, Any]:
+def transformer_wikitext2_benchmark(sagemaker_backend: bool = False, **kwargs):
     if sagemaker_backend:
         instance_type = DEFAULT_GPU_INSTANCE_1GPU
     else:
-        # For local backend, GPU cores serve different workers, so we
-        # need more memory
+        # For local backend, GPU cores serve different workers
         instance_type = DEFAULT_GPU_INSTANCE_4GPU
-    return {
-        "max_resource_level": 40,
-        "instance_type": instance_type,
-        "num_workers": 4,
-        "dataset_path": "./",
-    }
-
-
-def transformer_wikitext2_benchmark(sagemaker_backend: bool = False, **kwargs):
-    params = transformer_wikitext2_default_params(sagemaker_backend)
-    config_space = dict(
-        _config_space,
-        input_data_dir=params["dataset_path"],
-        **{MAX_RESOURCE_ATTR: params["max_resource_level"]},
+    fixed_parameters = dict(
+        **{MAX_RESOURCE_ATTR: 40},
+        d_model=256,
+        ffn_ratio=1,
+        nlayers=2,
+        nhead=2,
+        bptt=35,
+        optimizer_name="sgd",
+        input_data_dir="./",
+        use_cuda=1,
+        seed=1111,
+        precision="float",
+        log_interval=200,
```

Comment:
Where do these params come from?

Reply:
They are default values in the arg parser in the script. This just makes them more accessible, and shows that these ones can also be set.